WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
Ok, lets get the auto scrolling working and put the two buttons side by side on mobile. I'd also like to understand why the header doesn't float on mobile (which is a good thing, but I want to understand why!)